apps/http-api-app/src/main/resources/http_api_app.conf

```diff
@@ -28,10 +28,10 @@ application {
         includeRestApi = true
         server = {
             protocol = "http://"
-            host = "localhost"
+            host = "0.0.0.0"
             port = 8090
         }
-        localhostOnly = true    // Accessible only via localhost
+        localhostOnly = false    // Accessible only via localhost, default allow LAN
```

Comment:
Also here I would leave it to a JVM argument.

Users who run a headless node will be able to pass that argument.